controllers/GuidelineController.js

```diff
@@ -1,8 +1,18 @@
 const fs = require("fs");
 const marked = require("../util/Markdown");
+const path = require("path");
 
 exports.getGuidelines = (req, res, next) => {
-    fs.readFile(__dirname + "/../config/community_guidelines.md", "utf8", (err, data) => {
+    const guidelinesLocalePath = path.join(process.cwd(), "config", `community_guidelines_${res.locale}.md`);
```

Comment:
Can we move community guidelines out of the config folder if we're going to follow this pattern